### ISSUE TITLE
[REQ] Recheck FFs when launching a roblox window.

### ISSUE THREAD

**Issue by KorosAzolus** [bloxstraplabs/bloxstrap#1865]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Bloxstrap only checks FFs on the first window launch. It doesn't check for updates when opening other windows until all windows have been closed. Without Bloxstrap I can just change my FFs manually and it'll apply on all subsequent instances.

I like this program but this is a deal breaker for me. :(

Tldr: Bloxstrap prevents FF's from applying when opening a new window without closing all windows.

**Comment by pizzaboxer** [comment]
Planned to be supported in the future
